### PR TITLE
Readme instructions to refresh code excerpts

### PR DESCRIPTION
**Pull request by khanhnwin**
_Description of what this PR is changing or adding, and why:_ Add instructions for fixing broken build due to out of sync code excerpts.

_Issues fixed by this PR (if any):_

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,6 +268,14 @@ You can deploy your local edits to a personal staging site as follows.
 
    Your personal version of the Flutter website is now deployed to Firebase. 
    Copy the serving URL from the command output.
+   
+## Refreshing code excerpts
+
+If the build is failing with the following error: `=> ERROR: some code excerpts were not refreshed!` this means that some code excerpt(s) in the markdown files don't match up with the code in its correlated dart file.
```
Comment (sfshaza2) [inline README.md:274]:
```suggestion

A build that fails with the error `=> ERROR: some code excerpts were not refreshed!`
means that one or more code excerpts in the markdown file aren't identical to the
code in the corresponding `.dart` file. 
```

File: README.md [hunk 2]
```diff
@@ -268,6 +268,14 @@ You can deploy your local edits to a personal staging site as follows.
 
    Your personal version of the Flutter website is now deployed to Firebase. 
    Copy the serving URL from the command output.
+   
+## Refreshing code excerpts
+
+If the build is failing with the following error: `=> ERROR: some code excerpts were not refreshed!` this means that some code excerpt(s) in the markdown files don't match up with the code in its correlated dart file. 
+
+To resolve this error, from the root of the `website` directory, run `./tool/refresh-code-excerpts.sh`.
```
Comment (sfshaza2) [inline README.md:276]:
```suggestion
To resolve this error, from the root of the `website` directory,
run `./tool/refresh-code-excerpts.sh`. 
```

**Review by sfshaza2 — APPROVED** [review]
lgtm, with suggested edits

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Comment by khanhnwin** [comment]
Thanks @sfshaza2!

**Pull request merged**